### PR TITLE
Script for root transition.

### PR DESCRIPTION
**Pull request by myidpt**

### CODE REVIEW

**Comment by incfly** [comment]
`security/tools/` might be better? `bin` now contains one single outdated integration test script

File: security/tools/root_transition/root-transition.sh [hunk 1]
```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
```
Comment (wenchenglu) [inline security/tools/root_transition/root-transition.sh]: add instructions how to run this script on the top?

Comment (myidpt) [reply security/tools/root_transition/root-transition.sh]: Done in another PR.

**Comment by incfly** [comment]
might worth adding a separate `OWNER` file to only one or two people.

**Pull request closed without merging**